tests/log/Log.cpp

```diff
@@ -71,9 +111,41 @@ TEST_CASE("Testing the Log<>() function", "[api][log]") {
     NUClear::PowerPlant plant(config);
 
     // We are installing with an initial log level of debug
```

Comment:
Are you?

Reply:
Technically yes, but I'm gonna remove the comment since it's irrelevant